tasklist/docker-compose.yml

```diff
@@ -218,7 +218,7 @@ services:
     environment:
       - camunda.tasklist.elasticsearch.url=http://elasticsearch:9200
       - camunda.tasklist.zeebeElasticsearch.url=http://elasticsearch:9200
-      - camunda.tasklist.zeebe.gatewayAdress=zeebe:26500
+      - camunda.tasklist.zeebe.gatewayAddress=zeebe:26500
```

Comment:
same here given the typo the config should have had no effect, @camunda/human-task-orchestration-dev I wonder how this was working for you?